### ISSUE TITLE
USB Audio Adapter for microphone 

### ISSUE THREAD

**Issue by tommyna5** [alexa/alexa-avs-sample-app#738]
can I use USB Audio Adapter instead of the microphone? do I have to change anything from the settings?

![](http://geeksvalley.com/wp-content/uploads/2015/10/white-sound-card.jpg)
Thank you

**Comment by HydroCarbons** [comment]
Yes; more details here _https://learn.adafruit.com/usb-audio-cards-with-a-raspberry-pi_

**Comment by elstryan** [comment]
Hi, 
Did HydroCarbon's suggestion work for you?

**Comment by mmccool** [comment]
I have used exactly this adapter (in fact, to get AVS running with an Intel-based UP Board instead of a RaspPi).   I suggest changing the defaults for your system globally (you can still override them if you want to use a different device for some application, but global defaults will lead to the fewest hassles...).
